docs/devices/ZYCT-202.md

```diff
@@ -23,6 +23,10 @@ Factory reset the remote by holding the 0 (off) button for +-20 seconds.
 To establish a connection keep the remote within 2 meters from the hub.
 Press and hold the smart group button (button with two bulbs) and wait until the lights, below the channels, flash.
 
+### Binding
+Currently, binding to a device or group is not possible from zigbee2mqtt
```

Comment:
I think this should be rephrased to something like: "This device does not support binding". Now it looks like it's a Zigbee2MQTT issue.

Reply:
I understand...
but, technically zigbee2mqtt doesn't support it, you can bind with touchlink according the manual

Reply:
Those should be the same bind commands as: https://www.zigbee2mqtt.io/information/binding.html

Reply:
I'm sorry but binding dins't work...
I already sent the device back to the store, so i can't test anymore